src/mixins/filing-template-mixin.ts

```diff
@@ -98,11 +98,7 @@ export default class FilingTemplateMixin extends DateMixin {
   @Action(useStore) setRestorationCourtOrder!: ActionBindingIF
   @Action(useStore) setRestorationExpiry!: ActionBindingIF
   @Action(useStore) setRestorationType!: ActionBindingIF
-  @Action(useStore) mutateRestorationApprovalType!: ActionBindingIF
-  @Action(useStore) mutateRestorationCourtOrder!: ActionBindingIF
-  @Action(useStore) mutateRestorationExpiry!: ActionBindingIF
-  @Action(useStore) mutateRestorationType!: ActionBindingIF
-  @Action(useStore) mutateRestorationRelationships!: ActionBindingIF
+  @Action(useStore) setRestorationRelationships!: ActionBindingIF
```

Comment:
Former methods that begin, "mutate..." have been renamed to "set..." during the upgrade to Pinia.